Makefile

```diff
@@ -560,7 +569,6 @@ clean-examples: ## Remove all the temporary files generated in the examples fold
 
 clean-e2e:
 	rm -f $(E2E_CONF_FILE_ENVSUBST)
-	rm -rf $(E2E_ENVSUBST_DIR)
```

Comment:
Why is this removed?

Reply:
Thanks, the line `rm -f $(E2E_CONF_FILE_ENVSUBST)` removed instead of `rm -rf $(E2E_ENVSUBST_DIR)` since it is a child file inside the Dir